en/guides/user-management/README.md

```diff
@@ -0,0 +1,127 @@
+
+::: tip
+
+You can see that you are a [teacher](#teacher) when you can see the **management menu** (menu <span class="guilabel">Admin</span>) left of the [user menu](/en/for-students#user-menu) at the right side of the [navigation drawer](/en/for-students#navigation-drawer).
+
+![image](./staff.admin_menu.en.png)
+:::
+
+::: tip Important
+
+We divide the [enrolled](/en/for-students#course-register) members of a [course](/en/course-management#course) in [students](#student) and
+[course managers](/en/course-management#course-manager) depending on the [management permissions](#management-permissions)they have been granted for the course. Only course managers can manage a course. The 
+[managing](/en/course-management#managing-course-users) of 
+[course users](/en/course-management#course-user) and [appointing](/en/course-management#appointing-course-managers) course managers is discussed in [course management](/en/course-management).
+:::
+
+## Navigating to a user
+
+As a [teacher](#teacher), you navigate to the [user overviewt](#user-overview) by selecting <span class="guilabel">Users</span> in the [management menu](#management-menu) at the right side of the [navigation panel](/en/for-students#navigation-panel).
+
+![admin menu users](./staff.admin_menu_users.en.png)
+
+## User overview
+In the **user overview**, the user's username, real name, email and [progress statistics](#user-progress-statistics) are listed from the [user profile](/en/for-students#user-profile) from all users. In the first column, you can find an [icon](#user-permissions-icon) that corresponds to the [management permissions](#management-permissions) that have been [assigned](#assign-permissions) to that user.
+
+![image](./staff.users.en.png)
+
+## Searching a user
+Use the search bar at the top of the [user overview](#user-overview) to search for specific users based on their usernames, names or email addresses.
+
+![image](./staff.users_filtered.en.png)
+
+## Selecting a user
+Click on the username to [navigate](#navigating-to-a-user) to the [profile page](/en/for-students#profile-page) of te user.
+
+![image](./staff.users_filtered_link.en.png)
+
+## User progress statistics voortgangsstatistieken
+The **progress statistics** of a user consists of two numbers $c/s$, where $s$ is the number of [exercises](/en/for-students#exercise) for which that user has [submitted](/en/for-students#submitting-a-solution) atleast one [solution](/en/for-students#solution) and $c$ is the number of [exercises](/en/for-students#exercise) for which that user has [submitted](/en/for-students#submitting-a-solution) atleast one *correct* [solution](/en/for-students#solution).
+
+
+## Editing a user profile
```

Comment:
This is not possible anymore for teachers, so I think this section can be removed.